Library/Homebrew/cask/artifact/pkg.rb

```diff
@@ -65,7 +65,26 @@ def run_installer(command: nil, verbose: false, **_options)
             "USER"     => User.current,
             "USERNAME" => User.current,
           }
-          command.run!("/usr/sbin/installer", sudo: true, args: args, print_stdout: true, env: env)
+          result = command.run(
+            "/usr/sbin/installer",
+            sudo:         false,
+            args:         args,
+            print_stdout: true,
+            env:          env,
+            must_succeed: false,
```

Comment:
```suggestion
```

This is already the default.

Reply:
Wanted to be explicit here, to emphasize what's being done.